### PR TITLE
feat: use markdown as blog and changelog instead of ghost

### PR DESCRIPTION
**Pull request by boojack**
Click here to [Preview](https://bytebase-com-git-steven-con-140-display-markdow-36e297-bytebase.vercel.app/).

### CODE REVIEW

File: common/utils.ts [hunk 1]
```diff
@@ -15,3 +15,14 @@ export const getBrowserOSName = (): string => {
...
+export const calcReadingTime = (content: string): string => {
+  const wordsPerMinute = 200;
+  const wordAmount = content.split(" ").length;
+  if (wordAmount <= 200) {
+    return "less than 1 min read";
+  }
+
+  const count = Math.ceil(wordAmount / wordsPerMinute);
+  return `${count} min read`;
+};
```
Comment (LiuJi-Jim) [inline common/utils.ts:28]: Interesting! Might be better to implement a similar algorithm for Chinese in the future.

File: pages/blog/_slug/index.vue [hunk 2]
```diff
@@ -1,78 +1,92 @@
...
+      ),
+    };
+
+    return {
+      blog,
+    };
   },
   head() {
-    const post = (this as any).post;
+    const blog = (this as any).blog;
```
Comment (LiuJi-Jim) [inline pages/blog/_slug/index.vue:81]: This seems a little bit flaky here. Didn't nuxt.js provide any type infer helpers for `this` here?

**Review by LiuJi-Jim — APPROVED** [review]
LGTM for the `.ts` and `.vue` files.

File: pages/blog/_slug/index.vue (hunk 2 above)
Comment (boojack) [reply pages/blog/_slug/index.vue:81]: It's a vue2 coding style, maybe typescript just doesn't know what is `this`. Should I need to install some extension in VScode?

Comment (LiuJi-Jim) [reply pages/blog/_slug/index.vue:81]: I see `defineNuxtComponent` from nuxt 3 [here](https://v3.nuxtjs.org/api/utils/define-nuxt-component#definenuxtcomponent) which looks really like `defineComponent` from vue but it's not yet documented. T_T

**Review by tianzhou — APPROVED** [review]
LGTM

**Comment by tianzhou** [comment]
🚀🚀🚀🚀🚀

**Pull request merged**